docs/en_US/Tuner/BuiltinTuner.rst

```diff
@@ -221,11 +156,6 @@ As a strategy in a Sequential Model-based Global Optimization (SMBO) algorithm,
 PBT Tuner
 ^^^^^^^^^
 
-..
-
-   Built-in Tuner Name: **PBTTuner**
-
-
 **Suggested scenario**
 
 Population Based Training (PBT) bridges and extends parallel search methods and sequential optimization methods. It requires relatively small computation resource, by inheriting weights from currently good-performing ones to explore better ones periodically. With PBTTuner, users finally get a trained model, rather than a configuration that could reproduce the trained model by training the model from scratch. This is because model weights are inherited periodically through the whole search process. PBT can also be seen as a training approach. If you don't need to get a specific configuration, but just expect a good model, PBTTuner is a good choice. `See details <./PBTTuner.rst>`__
```

Comment:
we have two types of hyper link: "Detailed Description" and "See details"

Reply:
Seems not a big problem. Don't want to touch them now.